src/components/POLine/Cost/CostForm.js

```diff
@@ -32,7 +32,7 @@ export const requiredPositiveQuantity = (value) => {
 };
 
 const validateRequiredNotNegative = (value) => {
-  return value === 0 || value > 0
+  return value === 0 || value >= 0
```

Comment:
`=` here should be covered in first condition, `value === 0`. IN other case it covers case when `'' == 0`, that I don't think is a right thing.

Reply:
I did not see the conditions)) My changes do not make any sense) This validation work fine without any changes